sdk/keyvault/azure-security-keyvault-secrets/src/secret_serializers.cpp

```diff
@@ -236,3 +238,22 @@ std::string KeyVaultSecretPropertiesSerializer::KeyVaultSecretPropertiesSerializ
 
   return payload.dump();
 }
+
+std::vector<uint8_t> KeyvaultBackupSecretSerializer::KeyvaultBackupSecretDeserialize(
+    Azure::Core::Http::RawResponse const& rawResponse)
+{
+  auto const& body = rawResponse.GetBody();
+  auto jsonParser = json::parse(body);
+  auto encodedResult = jsonParser[_detail::ValuePropertyName].get<std::string>();
```

Comment:
For Key Vault Keys, we decode the result from Base64.  
See: sdk/keyvault/azure-security-keyvault-keys/src/key_backup.cpp

Then we encoded it back for Serializing.  I think this gives a better UX to customers as they don't need to worry about Decoding/Encoding

Reply:
are you sure ? i'm not 100% but i think .net does not decrypt it

Reply:
I am sure we are doing it for Backup/Restore keys, @heaths is it not required for secrets?

Reply:
Yes, here's NET:

https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Secrets/src/SecretBackup.cs#L17

Just like we do with Keys